SSA/Projects/FullyHomomorphicEncryption/Basic.lean

```diff
@@ -534,44 +590,37 @@ theorem R.coeff_fromPoly {q n : ℕ} [Fact (q > 1)] (p : (ZMod q)[X]) : R.coeff
   have H := R.representative_fromPoly_toFun (a := p) (n := n)
   norm_cast at H ⊢
 
-
-/-- since `0` is the additive identity, getting from `(x :: xs)` at index `i` with -/
-theorem List.getD_snoc_zero (x : ℤ) (xs : List ℤ) :
-  (xs ++ [x]).getD i 0 = (xs.getD i 0) + (if i == xs.length then x else 0) := by sorry
-
+/- 1. given a list of coefficients, building the ring element and then picking the repr from
```

Comment:
this was our map of the proof, I guess we don't need it anymore? (seems a bit confusing here with context)